### ISSUE TITLE
linux-4.8.12-3 can be  installed without being blocked due to missing archzfs packages

### ISSUE THREAD

**Issue by UweSauter** [archzfs/archzfs#89]
I was able to  update the kernel to 4.8.12-3 without being blocked by missing ZFS packages for that version. It seems that the dependencies between packages "linux" and "spl-linux-git / spl-utils-linux-git / zfs-linux-git / zfs-utils-linux-git" were removed somehow.

Now I have a system that doesn't autoload the modules (manually insmod'ing the modules from 4.8.12-2 works).

**Comment by johnramsden** [comment]
Same thing happened to me. Had to chroot in and downgrade.

**Comment by farnoy** [comment]
Same thing here, I had to `insmod` manually on liveusb, then arch-chroot and downgrade.

**Comment by UweSauter** [comment]
Not sure if this is the cause but I had a look into the "desc" files  you can find in the repository database file [1] and there the packages only depend on linux-4.8.12 (without build number). But the modules were installed into /lib/modules/4.8.12-2-ARCH/extra/ where 4.8.12-3 doesn't look.


[1] http://demizerone.com/archzfs/x86_64/archzfs.db.tar.xz

**Comment by UweSauter** [comment]
New packages are out. Thanks!
But in the "desc" files there still is just "linux-4.8.12" as dependency. Don't know if this can be solved at all.

From zfs-linux-git-0.7.0_rc2_r78_gd57f03e_4.8.12_3-1/desc out of archzfs.db.tar.xz:

%DEPENDS%
kmod
spl-linux-git
zfs-utils-linux-git
linux=4.8.12

%MAKEDEPENDS%
linux-headers=4.8.12
git

**Comment by demizer** [comment]
This is a problem with kernel pkgrel bumps: 4.9.7-1 -> 4.9.7-2 and the kernel module path being incorrect (thanks @UweSauter for the explanation).
 
This issues will be eliminated when archzfs moves to use extramodules (like the nvidia package).

**Comment by demizer** [comment]
Duplicate of #87
